src/runmode-pfring.c

```diff
@@ -370,6 +373,44 @@ void *ParsePfringConfig(const char *iface)
         }
     }
 
+    if (ConfGetChildValueWithDefault(if_root, if_default, "copy-iface", &out_interface) == 1) {
+        if (strlen(out_interface) > 0) {
+            pfconf->out_interface = out_interface;
+        }
+    }
+
+    if (ConfGetChildValueWithDefault(if_root, if_default, "copy-mode", &copymodestr) == 1) {
+        if (pfconf->out_interface == NULL) {
+            SCLogInfo("Copy mode activated bu no destination"
+                      " iface. Disabling feature");
+        } else if (strlen(copymodestr) <= 0) {
+            pfconf->out_interface = NULL;
+        } else if (strcmp(copymodestr, "ips") == 0) {
+            SCLogInfo("PF_RING IPS mode activated %s->%s",
+                    iface,
+                    pfconf->out_interface);
+            pfconf->copy_mode = PFRING_COPY_MODE_IPS;
+        } else if (strcmp(copymodestr, "tap") == 0) {
+            SCLogInfo("PF_RING TAP mode activated %s->%s",
+                    iface,
+                    pfconf->out_interface);
+            pfconf->copy_mode = PFRING_COPY_MODE_TAP;
+        } else {
+            SCLogInfo("Invalid mode (not in tap, ips)");
```

Comment:
I think all errors in this code should just that: errors. This is an issue in the afpacket code as well.